pkg/querier/blocks_store_queryable.go

```diff
@@ -135,6 +137,11 @@ func newBlocksStoreQueryableMetrics(reg prometheus.Registerer) *blocksStoreQuery
 			Name: "cortex_querier_blocks_with_compactor_shard_but_incompatible_query_shard_total",
 			Help: "Blocks that couldn't be checked for query and compactor sharding optimization due to incompatible shard counts.",
 		}),
+		// Named to be consistent with distributor_query_ingester_chunks_total
```

Comment:
This metrics is very similar to, and is named to be consistent with, the metrics in pkg/querier/stats/query_metrics.go, but those are mostly specific to ingesters/distributors and not as easy to share with a block store queryable.